src/index.css

```
@@ -0,0 +1,260 @@
+}
+
+.about__user {
+    width: 340px;
+    display: flex;
+    align-items: center;
+    margin:  0px 30px 0px 0px;
+}
+
+.about__user>:nth-child(2) {
+    padding: 0px 0px 0px 30px;
+}
+
+.txt {
+    color: #fff;
+    font-size: 16px;
+    letter-spacing: 2px;
+}
+.main__txt {
+    color: #fff;
+    font-size: 32px;
+    font-weight: 700;
+    letter-spacing: 2px;
+}
+.icon__u {
+    border-radius: 100%;
+    width: 60px;
+    height: 60px;
+}
+
+.Logo {
+    width: 220px;
+    align-items: center;
+}
+
+.player {
+    width: 1000px;
+    height: 140px;
+    background-color: rgba(19, 18, 18, 0.3);
+    backdrop-filter: blur(20px);
+    border-top-left-radius: 30px;
+    border-top-right-radius: 30px;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+}
+
+.move__player
+ {
+    position: fixed;
+    bottom: 0;
+    width: inherit;
+    display: flex;
+    justify-content: center;
+     
+ }
+
+.button__player {
+    width: 690px;
+    align-items: center;
+    padding: 20px 0px 10px 0px;
+}
+
+.music__line {
+    width: 680px;
+    padding: 0px 0px 15px 0px;
+}
+
+.flex__between {
+    display: flex;
+    justify-content: space-between;
+}
+
+.play__button__banner {
+    width: 120px;
+    padding: 5px 20px;
+    border-radius: 25px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    background-color: white;
+    border: #fff;
+}
+
+.big__banner__inf {
+    width: 300px;
+    height: 160px;
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-items: flex-start;
+    margin: 50px;
+}
+
+.big__banner {
+    width: 980px;
+    height: 400px;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    flex-direction: row;
+    align-items: center;
+    background-image: url(../resources/banner/banner__background.png);
+    border-radius: 10px;
+    margin: 0px 10px 0px 0px;
+}
+
+.txt__black {
+    color: #000;
+}
+
+.txt__24 {
+    font-size: 24px;
+    letter-spacing: 2px;
+}
+
+.playlist__podcast {
+    width: 300px;
+    height: 400px;
+    background: #131212;
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column;
+    text-decoration: none;
+    margin: 0px 10px 20px 0px;
+
+}
+
+.banner,
+.recomendations {
+    width: calc(400px + (1660 - 400) * ((100vw - 500px) / (1920 - 500)));
+    height: auto;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between; 
+}
+
+.playlist__txt__block {
+    width: 170px;
+    height: 90px;
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-items: flex-start;
+    margin: 40px 0px 0px 20px;    
+}
+
+.txt__14 {
+    font-size: 14px;
+    color: #fff;
+    letter-spacing: 1px;
+}
+
+.flex__around__playlist {
+    display: flex;
+    justify-content: space-around;
+}
+
+.albums {
+    height: inherit;
+    margin: 0px 0px 0px 0px;
```

избыточная запись
Можно сократить до
```suggestion
    margin: 0px;
```

Также это избыточно и можно удалить